src-v2/Judge/NTS.Judge/Adapters/Behinds/DashboardBehind.cs

```diff
@@ -63,6 +84,68 @@ async Task CreateOfficials(IEnumerable<Domain.Setup.Entities.Official> setupOffi
         }
     }
 
+    async Task CreateParticipations(Domain.Setup.Entities.Event setupEvent) 
+    {
+        foreach(var competition in setupEvent.Competitions)
+        {
+            var setupPhases = competition.Phases;
+            var competitionDistance = 0m;
+            var phases = new List<Phase>();
+            var categoryEntriesPairs = new Dictionary<AthleteCategory, List<RankingEntry>>
+            {
+                { AthleteCategory.Senior, new List<RankingEntry>() },
+                { AthleteCategory.Children, new List<RankingEntry>() },
+                { AthleteCategory.JuniorOrYoungAdult, new List<RankingEntry>() },
+                { AthleteCategory.Training, new List<RankingEntry>() }
+            };
+            foreach(var phase in setupPhases)
+            {
+                phases.Add(new Phase(phase.Loop!.Distance, phase.Recovery, phase.Rest, competition.Ruleset, setupPhases.Last() == phase, competition.CriRecovery));
+                competitionDistance += (decimal)phase.Loop!.Distance;
+            }
+            foreach(var contestant in competition.Contestants)
+            {
+                var combination = contestant.Combination;
+                var maxSpeed = (double?)null;
+                var minSpeed = (double?)null;
+                //consider adding magic ints to StaticOptions or constants
+                if(competition.Type == CompetitionType.Qualification)
+                {
+                    minSpeed = 10;
+                    maxSpeed = 16;
+                }
+                if(combination.Athlete.Category == AthleteCategory.Children)
+                {
+                    minSpeed = 8;
+                    maxSpeed = 12;
+                }
+                maxSpeed = contestant.MaxSpeedOverride != null ? contestant.MaxSpeedOverride : maxSpeed;
+                var tandem = new Tandem(combination.Number, combination.Athlete.Person, combination.Horse.Name, competitionDistance, combination.Athlete.Country, combination.Athlete.Club, minSpeed, maxSpeed);
+                var participation = new Participation(competition.Name, competition.Ruleset, tandem, phases);
+                await _participationRepository.Create(participation);
```

Comment:
It's good idea to separate the domain creation logic and storage. Maybe something like StartFactory, DashboardFactory or CoreFactory class, which contains: `CreateParticipation`, `CreateRanking`, `CreateOfficial` etc